### PR TITLE
#60 - authentication views

### LINKED ISSUE #60: Authentication views
Simple views for login and registration

### PR DESCRIPTION
**Pull request by JakubZakowicz**
I have created views for login and register page

- login page
login_page

- register page
register_page

- after successfull registration
registered_page

This should close #60

### CODE REVIEW

File: resources/views/meetups/edit.blade.php [hunk 1]
```diff
@@ -1,43 +1,102 @@
...
-            <h1>Edit Meetup</h1>
-            @auth
-                <form method="post" action="{{ route('meetups.update', $meetup) }}">
-                    @method('PUT')
-                    @csrf
+    <div class="container md:w-[800px] mx-auto">
+        @auth
+            <form action="{{ route('meetups.update', $meetup) }}" method="post"
+                class="bg-white p-6 mt-20 rounded-20 shadow-xl" enctype="multipart/form-data">
+                @method('PUT')
```
[the remark this answers is not in the document]
Comment (mtracz) [reply resources/views/meetups/edit.blade.php:8]: Regarding to https://laravel.com/docs/9.x/blade#method-field, there is no question.

File: resources/views/home.blade.php [hunk 2]
```diff
@@ -0,0 +1,104 @@
+@extends('layouts.app')
+
+@section('content')
+    <div>
+        <div class="relative">
+            <div class="absolute inset-x-0 bottom-0 h-1/2 bg-gray-100"></div>
+            <div class="max-w-7x mx-auto ">
+                <div class="relative shadow-xl sm:overflow-hidden">
+                    <div class="absolute inset-0">
+                        <img class="h-full w-full object-cover" src={{ asset('static/images/hero.png') }}
```
[the remark this answers is not in the document]
Comment (MichalMyskow) [reply resources/views/home.blade.php:10]: All static images or uploaded images too?

File: src/Models/Meetup.php [hunk 3]
```diff
@@ -35,11 +36,17 @@ class Meetup extends Model
         "date",
         "place",
         "language",
+        "logo",
```
[the remark this answers is not in the document]
Comment (MichalMyskow) [reply src/Models/Meetup.php]: Should I create a new migration for renaming a column or edit old migration when application is not in production?

[the remark this answers is not in the document]
Comment (mtracz) [reply src/Models/Meetup.php]: Until it is not merged and is not in production, I prefer edit migration than create a new migration.

File: resources/views/home.blade.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (mtracz) [reply resources/views/home.blade.php:10]: From my point of view just static images - it is a suggestion for little optimalization, cause webp images have almost the same quality as other formats and take less size.. Converting uploaded images may be overkill.

**Review by mtracz — APPROVED** [review]

**Review by MichalMyskow — APPROVED** [review]

**Pull request merged**